src/20240528_Multi_GHOCrossChainLaunch/AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol

```diff
@@ -0,0 +1,316 @@
+      type(UpgradeableBurnMintTokenPool).creationCode,
+      abi.encode(
+        ghoToken, // token
+        CCIP_ARM_PROXY, // armProxy
+        false // allowlistEnabled
+      )
+    );
+
+    // proxy deploy and init
+    bytes memory tokenPoolInitParams = abi.encodeWithSignature(
+      'initialize(address,address[],address)',
+      GovernanceV3Arbitrum.EXECUTOR_LVL_1, // owner
+      new address[](0), // allowList
+      CCIP_ROUTER // router
+    );
+    bytes memory creationCode = abi.encodePacked(
+      type(TransparentUpgradeableProxy).creationCode,
+      abi.encode(
+        imple, // logic
+        MiscArbitrum.PROXY_ADMIN, // proxy admin
+        tokenPoolInitParams // data
+      )
+    );
+
+    (imple, proxy) = _deploy({
+      impleSalt: CCIP_TOKEN_POOL_IMPL_DEPLOY_SALT,
+      impleCreationCode: impleCreationCode,
+      proxySalt: CCIP_TOKEN_POOL_DEPLOY_SALT,
+      proxyCreationCode: creationCode
+    });
+  }
+}
+
+/**
+ * @title GHO Cross-Chain Launch
+ * @author Aave Labs
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x2a6ffbcff41a5ef98b7542f99b207af9c1e79e61f859d0a62f3bf52d3280877a
+ * - Discussion: https://governance.aave.com/t/arfc-gho-cross-chain-launch/17616
+ * @dev This payload consists of the following set of actions:
+ * 1. Deploy GHO
+ * 2. Deploy BurnMintTokenPool
+ * 3. Accept ownership of CCIP TokenPool
+ * 4. Configure CCIP TokenPool
+ * 5. Add CCIP TokenPool as GHO Facilitator
+ * 6. List GHO in Aave V3 Pool
+ * 7. Seed Aave Pool
+ */
+contract AaveV3Arbitrum_GHOCrossChainLaunch_20240528 is AaveV3PayloadArbitrum {
+  address public immutable GHO;
+  address public immutable GHO_IMPL;
+  address public immutable CCIP_TOKEN_POOL;
+  address public immutable CCIP_TOKEN_POOL_IMPL;
+
+  constructor() {
+    // Predict GHO contract address
+    GHO_IMPL = ICreate3Factory(MiscArbitrum.CREATE_3_FACTORY).predictAddress(
+      GovernanceV3Arbitrum.EXECUTOR_LVL_1,
+      Utils.GHO_IMPL_DEPLOY_SALT
+    );
+    GHO = ICreate3Factory(MiscArbitrum.CREATE_3_FACTORY).predictAddress(
+      GovernanceV3Arbitrum.EXECUTOR_LVL_1,
+      Utils.GHO_DEPLOY_SALT
+    );
+    // Predict CCIP TokenPool contract address
+    CCIP_TOKEN_POOL_IMPL = ICreate3Factory(MiscArbitrum.CREATE_3_FACTORY).predictAddress(
+      GovernanceV3Arbitrum.EXECUTOR_LVL_1,
+      Utils.CCIP_TOKEN_POOL_IMPL_DEPLOY_SALT
+    );
+    CCIP_TOKEN_POOL = ICreate3Factory(MiscArbitrum.CREATE_3_FACTORY).predictAddress(
+      GovernanceV3Arbitrum.EXECUTOR_LVL_1,
+      Utils.CCIP_TOKEN_POOL_DEPLOY_SALT
+    );
+  }
+
+  /// @dev 6. List GHO in Aave V3 Pool
+  function newListings() public view override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: GHO,
+      assetSymbol: 'GHO',
+      priceFeed: 0x3c786e934F23375Ca345C9b8D5aD54838796E8e7,
```

Comment:
@miguelmtzinf With the v3 ethereum market pricing GHO on primary market (fixed $1) and the v3 arbitrum pricing GHO on secondary market price could any discrepancies arise? Wanted to understand a bit deeper on why we are pricing GHO on arbitrum using secondary market price over primary (fixed $1).

In case we go ahead and use the secondary market price for GHO, I think it is a good idea if we use the cap adapters for the feed: https://github.com/bgd-labs/aave-capo/blob/main/src/contracts/PriceCapAdapterStable.sol

Reply:
GHO price is fixed at $1 on Ethereum because GHO is being issued by the Aave Pool and acts as a stabilizing mechanism. GHO on Arbitrum Aave Pool works as any other listed asset.

This price feed is the Chainlink GHO/USD price oracle, VWAP across chains, so mainly using secondary market on Ethereum and CEXes atm

Reply:
Added capped gho / usd price feed in [0c2e4cc](https://github.com/bgd-labs/aave-proposals-v3/pull/350/commits/0c2e4ccefca54cbe567d70b31315ee89d405a519)

Reply:
But why from the Aave protocol perspective GHO should have a different price in Arbitrum than Ethereum? The question is not about the nature of being deposited vs minted, it is about the high-level implications.
For example, in which situation of only-borrowable GHO having 1:1 creates a problem.

Reply:
GHO works as a regular asset across chains, while specific features of GHO like price stability or backing are kept on Ethereum.

I do not see any problem on not using a fixed price on Arbitrum, specially taking into consideration it will not be collateral. The expansion of GHO to other networks and Aave Pools will be monitored and config can be adjusted by the DAO if needed.